apps/condo/pages/ticket/index.tsx

```diff
@@ -162,8 +161,7 @@ const TicketTable = ({
     const { getTrackingWrappedCallback } = useTracking()
     const timeZone = intl.formatters.getDateTimeFormat().resolvedOptions().timeZone
 
-    const auth = useAuth() as { user: { id: string } }
-    const user = get(auth, 'user')
+    const { user } = useAuth() as { user: { id: string } }
```

Comment:
```suggestion
    const { user } = useAuth()
```

looks like outdated code